lib/bdev/nvme/blockdev_nvme.c

```diff
@@ -390,6 +390,10 @@ probe_cb(void *cb_ctx, const struct spdk_nvme_probe_info *probe_info,
 		return false;
 	}
 
+	if (ctx->controllers_remaining > 0) {
```

Comment:
Actually, re-reading the conf parsing code, it looks like NumControllers is indeed intended to be used with BDF, although I'm not really sure why (I'll check with some folks to see if this is really needed or if a plain list of BDFs would be good enough).